docs/tutorials/zed-primer.md

```diff
@@ -1,67 +1,324 @@
-"time" type requires storing timestamps as strings. As a result, if read into
-`zq` as the strings they are, these `ts` values are not usable with
-time-specific operations in Zed, such as this attempt to use
-[time grouping]../zq/functions/bucket.md) to calculate total
-quantities shipped per day.
+Let's revert the most recent commit:
 
-```mdtest-command
-zq -z 'sum(quantity) by every(1d) | sort ts' shipments.ndjson
 ```
+zed revert 26i2N0uu6wEo5XAhPMid6eQsamF
+```
+=>
+```
+"main": 26i2N0uu6wEo5XAhPMid6eQsamF reverted in 26nY9AYOxx2WtSfKGjof9R2MOYb
+```
+
+We can run `count()` to see we're back to our original 100 values.
+
+```bash
+$ zed query 'count()'
+```
+=>
+```
+{count:100(uint64)}
+```
+
+If we made a mistake and we'd like to keep the data, we can also revert our 
+revert commit:
 
-#### Output:
-```mdtest-output
-{ts:error("every: time arg required"),sum:9158}
+```
+$ zed revert 26nY9AYOxx2WtSfKGjof9R2MOYb
 ```
 
-However, if we cast the `ts` field to the Zed `time` type, now the
-calculation works as expected.
+Running `count()` will show we're back to 2000 values:
 
-```mdtest-command
-zq -f table 'ts:=time(ts) | sum(quantity) by every(1d) | sort ts' shipments.ndjson
+```bash
+$ zed query 'count()'
+```
+=>
 ```
+{count:2000(uint64)}
+```
+
+## Running as a service
+
+Now that we compiled an interesting data set, how might we share this with 
+others? Using the `zed serve` command we can launch our Zed Lake as a service
+that will allow multiple clients to query and add data to the same lake. In a 
+separate console window run:
 
-#### Output:
-```mdtest-output
-ts                   sum
-2021-10-07T00:00:00Z 1432
-2021-10-08T00:00:00Z 1742
-2021-10-09T00:00:00Z 2980
-2021-10-10T00:00:00Z 3004
 ```
+$ zed serve
+```
+=>
+```
+{"level":"info","ts":1647957396.828584,"msg":"Open files limit raised","limit":10240}
+{"level":"info","ts":1647957396.8318028,"logger":"core","msg":"Started"}
+{"level":"info","ts":1647957396.83288,"logger":"httpd","msg":"Listening","addr":"[::]:9867"}
+```
+
+We now have a service running on `http://localhost:9867`. If we set the 
+`ZED_LAKE` environment variable we defined at the beginning to this URL we can
+run the full set of `zed` commands against this service:
+
+```
+$ export ZED_LAKE=http://localhost:9867
+$ zed query -Z 'min(created_at), max(created_at)'
+```
+=>
+```
+{
+    min: 2019-11-11T19:50:46Z,
+    max: 2021-08-10T19:48:56Z
+}
+```
+
+## Where to go from here?
+
+Obviously this is only the tip of the iceberg in terms of things can be done with
+the `zed` command. Some suggested next steps:
+
+1. Dig deeper into Zed Lakes by having a look at the [`zed` README](../zed/README.md).
+2. Get a better idea of ways you can query your data by looking at the 
+[Zed language documentation](../zq/language.md).
+
+If you have any questions or run into any snags join the friendly Zed community
+at the [Brim Data Slack channel](https://www.brimdata.io/join-slack/).
```

Comment:
```suggestion
at the [Brim Data Slack workspace](https://www.brimdata.io/join-slack/).
```

If being pedantic, they're called Slack workspaces, and then there's many channels within. :wink:

Reply:
Accurate!